sockeye/checkpoint_decoder.py

```diff
@@ -133,20 +132,19 @@ def decode_and_evaluate(self,
                                           models,
                                           vocab_source,
                                           vocab_target)
-        trans_wall_times = np.zeros((len(self.input_sentences),))
+        trans_wall_times = 0.0
+        translations = []
         with data_io.smart_open(output_name, 'w') as output:
             handler = sockeye.output_handler.StringOutputHandler(output)
-            translations = []
-            for i, input_sentence in enumerate(self.input_sentences):
-                tic = time.time()
-                trans_input = translator.make_input(i, input_sentence)
-                trans_output = translator.translate(trans_input)
+            tic = time.time()
+            trans_inputs = [translator.make_input(i, line) for i, line in enumerate(source_data)]
```

Comment:
I think there's a copy/paste error here. You should use `self.input_sentences ` no? That's why mypy is unhappy.

Reply:
Yes, should be fixed now

Reply:
have you run a quick seqcopy model with bleu monitoring to double check (as this is something not covered by the unit/integration tests)?